src/parseUtils.js

```diff
@@ -342,9 +342,10 @@ function getModulesLocations(node) {
       node.arguments[0].elements :
       node.elements;
 
-    return _.transform(modulesNodes, (result, moduleNode, i) => {
-      if (!moduleNode) return;
+    return modulesNodes.reduce((result, moduleNode, i) => {
+      if (!moduleNode) return result;
```

Comment:
Let's better inverse condition and return only once in the end of the function please.